User Guides/19-Restarting a Control Server Deployment.md

```diff
@@ -0,0 +1,22 @@
+# Restart a Failed Control Server
+The Control Server as the brain box of the entire system is still susceptible to failure. The following steps describes how to re-deploy a failed Control Server.
+
+#### Restart Control Server Deployment
+1. Take a note of your resource group name. 
+
+2. Login to [Azure](portal.azure.com)
+
+3. Locate your resource group
+
+4. Open the **Deployments** tab under your resource group.

```

Comment:
Image displays inline with text. Add 2 spaces at end of this line to force markdown line break

Reply:
Please do this for all images to force left align.

Reply:
Done